frontend/packages/console-dynamic-plugin-sdk/src/webpack/ConsoleRemotePlugin.ts

```diff
@@ -199,6 +203,18 @@ export class ConsoleRemotePlugin implements webpack.WebpackPluginInstance {
 
     compiler.options.output.publicPath = publicPath;
 
+    compiler.options.resolve = compiler.options.resolve ?? {};
+    compiler.options.resolve.alias = compiler.options.resolve.alias ?? {};
+
+    // Prevent PatternFly styles from being included in the compilation
+    getPatternFlyStyles().forEach((cssFile) => {
+      if (Array.isArray(compiler.options.resolve.alias)) {
+        compiler.options.resolve.alias.push({ name: cssFile, alias: false });
```

Comment:
`options.resolve.alias` is typically an object, but webpack also allows for an array representation.